### PR TITLE
doc: update CHANGELOG for v2.8.0

### PR DESCRIPTION
**Pull request by devbww**
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10979)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -127,10 +127,41 @@ the APIs in these libraries are stable, and are ready for production use.
...
+
+### [Bigtable](/google/cloud/bigtable/README.md)
+
+- fix: retries for GetIamPolicy, TestIamPermissions ([#10957](https://github.com/googleapis/google-cloud-cpp/pull/10957))
+- fix(bigtable): retries for CheckConsistency / AsyncWaitForConsistency ([#10955](https://github.com/googleapis/google-cloud-cpp/pull/10955))
+- docs(bigtable): clean up CreateTable sample ([#10844](https://github.com/googleapis/google-cloud-cpp/pull/10844))
+
+### [Cloud Asset](/google/cloud/asset/README.md)
+
+- feat(generator): prepare comment output for Fedora:37 ([#10749](https://github.com/googleapis/google-cloud-cpp/pull/10749))
```
Comment (coryan) [inline CHANGELOG.md:142]: I think I would not highlight this.

Comment (coryan) [reply CHANGELOG.md:142]: But I don't feel too strongly about it.

**Review by coryan — APPROVED** [review]
LGTM as-is, but maybe remove the stuff about google/cloud/asset/

File: CHANGELOG.md (hunk 1 above)
Comment (devbww) [reply CHANGELOG.md:142]: Right you are ... I removed this from other places, but missed this one.  Will do.

File: CHANGELOG.md [hunk 2]
```diff
@@ -127,10 +127,41 @@ the APIs in these libraries are stable, and are ready for production use.
 - [Alloy DB](/google/cloud/alloydb/README.md)
 - [API Keys](/google/cloud/apikeys/README.md)
 
+### [BigQuery](/google/cloud/bigquery/README.md)
+
+- fix: retries for GetIamPolicy, TestIamPermissions ([#10957](https://github.com/googleapis/google-cloud-cpp/pull/10957))
```
Comment (dbolduc) [inline CHANGELOG.md:132]: I would delete the library specific entries and leave the "Common Libraries" entry for this PR.... but your call.

**Review by dbolduc — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (devbww) [reply CHANGELOG.md:132]: SGTM.  Will do.

**Review by scotthart — APPROVED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/10979)* status: all files reviewed, 2 unresolved discussions (waiting on @coryan and @dbolduc)

File: CHANGELOG.md [hunk 3]
```diff
@@ -143,6 +153,11 @@ support clients for multiple versions of a GCP service from within the same
 library (e.g. `speech_v1::SpeechClient` and `speech_v2::SpeechClient`). See
 [#10170] for more details.
 
+- feat(datacatalog): generate lineage library ([#10977](https://github.com/googleapis/google-cloud-cpp/pull/10977))
```
Comment (dbolduc) [inline CHANGELOG.md:156]: argh, just noticed this should probably have its own entry

Comment (devbww) [reply CHANGELOG.md:156]: I thought not, given that it's only experimental.

Comment (dbolduc) [reply CHANGELOG.md:156]: I forgot that it was experimental. I still don't think it belongs under "Common Libraries". I think we would either delete the entry, or list it under "Data Catalog".

But given that we don't suspect `datacatalog` has any users... I don't care, and would be happy to save some builds.

**Review by dbolduc — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (devbww) [reply CHANGELOG.md]: Done (listed under "Data Catalog").

**Pull request merged**